tests/SprykerSdkTest/Spryk/Integration/Service/AddServiceFactoryTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdkTest\Spryk\Integration\Service;
+
+use Codeception\Test\Unit;
+
+/**
+ * Auto-generated group annotations
+ * @group SprykerSdkTest
+ * @group Spryk
+ * @group Integration
+ * @group Service
+ * @group AddServiceFactoryTest
+ * Add your own group annotations below this line
+ */
+class AddServiceFactoryTest extends Unit
+{
+    protected const SPRYK_NAME = 'AddServiceFactory';
```

Comment:
This is not required as the `tester` extracts the name from the test class name.